core/src/test/scala/org/apache/spark/memory/UnifiedMemoryManagerSuite.scala

```diff
@@ -163,18 +170,19 @@ class UnifiedMemoryManagerSuite extends MemoryManagerSuite with PrivateMethodTes
 
   test("storage does not evict execution") {
     val maxMemory = 1000L
+    val taskAttemptId = 0L
     val (mm, ms) = makeThings(maxMemory)
     // First, ensure the test classes are set up as expected
     val expectedStorageRegionSize = 500L
     val expectedExecutionRegionSize = 500L
-    val storageRegionSize = getStorageRegionSize(mm)
+//    val storageRegionSize = getStorageRegionSize(mm)
     val executionRegionSize = maxMemory - expectedStorageRegionSize
-    require(storageRegionSize === expectedStorageRegionSize,
-      "bad test: storage region size is unexpected")
+//    require(storageRegionSize === expectedStorageRegionSize,
```

Comment:
I don't think that we need these asserts anymore since these invariants are checked via assertions inside of the pool itself.